src/pretix/base/models/event.py

```diff
@@ -544,6 +546,59 @@ def get_users_with_permission(self, permission):
             Q(is_superuser=True) | Q(twp=True)
         )
 
+    @staticmethod
+    def has_paid_things(event):
+        item = apps.get_model('pretixbase', 'Item')
+        item_variation = apps.get_model('pretixbase', 'ItemVariation')
+        return item.objects.filter(event=event, default_price__gt=0).exists()\
+            or item_variation.objects.filter(item__event=event, default_price__gt=0).exists()
+
+    @staticmethod
+    def has_payment_provider(event):
```

Comment:
Same here, I see no reason to have this as a static method, this could be a ``@property`` just as well.